example/spec/ingestGranule/IngestUMMGSuccessSpec.js

```diff
@@ -168,6 +169,12 @@ describe('The S3 Ingest Granules workflow configured to ingest UMM-G', () => {
 
   afterAll(async (done) => {
     try {
+      // pre-delete: Remove the granule from CMR
+      await granulesApiTestUtils.removeFromCMR({
```

Comment:
Not a blocker, but seems like we could use a utility function here to "removePublishedGranule" or something like that. It could call the API lambda, but just might be cleaner and easier to use.

Reply:
I'm not sure what the difference would be between that and what we have now. Do you mean having a `removePublishedGranule` that does `removeFromCmr` and `deleteGranule`?

Reply:
Yeah that is what I was thinking so we're less likely to miss the remove from cmr step in the future.